### PR TITLE
docs: KB about Outlook alongside Fe

### PR DESCRIPTION
**Pull request by NickIliev**
Related to https://kinvey.atlassian.net/browse/FID-2486

### CODE REVIEW

File: knowledge-base/setup-fiddler-alongside-outlook.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+|   |   |
+|---|---|
+| Fiddler Everywhere | 1.0.0 and above |
+| 3rd-party application | Microsoft Outlook, Office365 |
+| Operating System | Windows, macOS |
+
+## Description
+
+Some third-party applications like **Microsoft Outlook** are not working seamlessly alongside MITM proxies like Fiddler Everywhere.  How can I handle this issue?
```
Comment (yordan-mitev) [inline knowledge-base/setup-fiddler-alongside-outlook.md:20]: Use regular font for the product names, see https://testdocs.telerik.com/style-guide/element-formatting.

File: knowledge-base/setup-fiddler-alongside-outlook.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Solution
+
+In some cases, the third-party implementation restricts the application usage when the traffic goes through the MITM proxy. In other cases, the applications are not handling well dynamic change of the system proxy. The desktop application **Outlook** is one example of an application that might stop working when Fiddler Everywhere acts as a system proxy. To resolve issues with Outlook not working when Fiddler is turned on, follow these steps:
```
Comment (yordan-mitev) [inline knowledge-base/setup-fiddler-alongside-outlook.md:24]: In the previous paragraph, you talk about "third-party applications like **Microsoft Outlook**", but now you switch to "third-party implementation" and I cannot grasp what the latter means in this context. Can you be more specific?

- are not handling well dynamic change of the system proxy -> are not handling well _the_ dynamic change of the system proxy
- Don't use bold for product names.

File: knowledge-base/setup-fiddler-alongside-outlook.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+- Exit the **Microsoft Outlook** application. Make sure it is not running in the background.
```
Comment (yordan-mitev) [inline knowledge-base/setup-fiddler-alongside-outlook.md:26]: - Don't use bold for product names.
- Exit the **Microsoft Outlook** application -> Close Microsoft Outlook.

File: knowledge-base/setup-fiddler-alongside-outlook.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    *office.com;*office365.com;*live.com;*outlook.com;autodiscover.progress.com
+    ```
+
+    ![bypassing Outlook endpoints in Fiddler](../images/kb/outlook-bypass.png)
+
+- Start the **Live traffic** capturing in Fiddler Everywhere.
+
+- Restart the **Microsoft Outlook** application.
+
+That's it! After the above, the traffic generated from Outlook will bypass the Fiddler proxy.
```
Comment (yordan-mitev) [inline knowledge-base/setup-fiddler-alongside-outlook.md:42]: After the above... -> After completing the steps above...

**Review by rosen-vladimirov — APPROVED** [review]
For the technical explanation

**Pull request merged**